caktus_primer/static/js/index.js

```diff
@@ -97,24 +70,17 @@ export class Dropdown extends React.Component {
             let className = ""
             if (i == index) className = "selected"
             return (
-                <li key={i} className={className} >{
-                    React.cloneElement(el, {
-                        "ref": (el) => {
-                            if (i == index) this.curLink = el
-                        }
-                    })
-                }</li>
+                <li key={i} className={className} onMouseUp={this.handleClickOutside}>{el}</li>
             )
         })
 
         return (
             <div className="primer-dropdown" ariaExpanded={this.state.expanded} ariaHaspopup="true"
                 onKeyDown={(ev)=>this.key(ev)}
```

Comment:
I'd suggest using `this.key.bind(this)` here! No real difference, but if you're creating an anonymous function of one argument that just passes that argument to another function of one argument, you could just be using that latter function directly (though you still have to use `bind` because we're in this sorta-OOP-but-pretending-to-be-FP setting).